File: app/src/main/java/com/example/secpass/ui/Add_PanCard.java
package com.example.secpass.ui;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.EditText;
import android.widget.ImageView;
import android.widget.Spinner;

import com.example.secpass.R;
import com.example.secpass.helper.MyDatabaseHelper;

public class Add_PanCard extends AppCompatActivity {
EditText etTitlepc,etNamepc,etPanNumberpc,etNotepc;
Spinner spnCategory;
Button btnSavepc;
ImageView imgClosepc;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_add__pan_card);

        etTitlepc=findViewById(R.id.etTitlepc);
        etNamepc=findViewById(R.id.etNamepc);
        etPanNumberpc=findViewById(R.id.etPanNumberpc);
        etNotepc=findViewById(R.id.etNotepc);
        btnSavepc=findViewById(R.id.btnSavepc);
        spnCategory=findViewById(R.id.spnCategory);
        imgClosepc=findViewById(R.id.imgClosepc);

        imgClosepc.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent i = new Intent(Add_PanCard.this,DashBoardActivity.class);
                startActivity(i);
            }
        });
        btnSavepc.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                MyDatabaseHelper myDB = new MyDatabaseHelper(Add_PanCard.this);
                myDB.adddpancard(etTitlepc.getText().toString().trim(),
                        etNamepc.getText().toString().trim(),
                        Integer.valueOf(etPanNumberpc.getText().toString().trim()),
                      "",
                        etNotepc.getText().toString().trim()
                );

            }
        });
    }
}

File: app/src/main/java/com/example/secpass/ui/Add_PanCard.kt
package com.example.secpass.ui

import android.content.Intent
import android.os.Bundle
import android.os.PersistableBundle
import android.view.View
import androidx.appcompat.app.AppCompatActivity
import com.example.secpass.R
import com.example.secpass.helper.MyDatabaseHelper
import kotlinx.android.synthetic.main.activity_add__pan_card.*
import kotlinx.android.synthetic.main.activity_add_election_card.*

class Add_PanCard : AppCompatActivity(), View.OnClickListener {

    override fun onCreate(savedInstanceState: Bundle?, persistentState: PersistableBundle?) {
        super.onCreate(savedInstanceState, persistentState)
        setContentView(R.layout.activity_add__pan_card)
        initview()
    }

    private fun initview() {
        btnSavepc.setOnClickListener(this)
        imgClosepc.setOnClickListener(this)
    }

    override fun onClick(v: View?) {
        when (v?.id) {
            R.id.btnSavepc -> {
                var db = MyDatabaseHelper(this)
                db.addidproof(
                    etTitlepc.text.toString(), etNamepc.text.toString(),
                    etPanNumberpc.text.toString(), "", "",
                    "", "", "", etNotepc.text.toString(), etTitle.text.toString(), ""
                )
                startActivity(Intent(this, DashBoardActivity::class.java))
            }
            R.id.imgClosepc -> {
                onBackPressed()
            }
            R.id.imgAccount -> {

            }

        }
    }

}
